Wordle.API/Wordle.API/Models/Player.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Wordle.Api.Models
+{
+    [Table("Player")]
+    public class Player
+    {
+        public int PlayerId { get; set; }
+        public string Name { get; set; } = "";
```

Comment:
```suggestion
        public string Name { get; set; } = null!;
```
This is what Grant was suggesting to do in class but yours works fine but just noting